lib/bundler/installer.rb

```diff
@@ -156,6 +157,24 @@ def install(options)
       install_in_parallel jobs, options[:standalone], force
     end
 
+    def ensure_specs_are_compatible!
```

Comment:
I wonder if this would be useful to generalize interface wise and allow it to take in a specific ruby version and rubygems version as arguments. `ensure_specs_are_compatible(ruby_version, rubygems_version)`